server/models/Team.ts

```diff
@@ -121,16 +121,20 @@ class Team extends ParanoidModel {
   }
 
   get url() {
+    // custom domain
```

Comment:
I believe this comment is correct - the `domain` field on a team represents a custom domain right @tommoor ?

Reply:
Correct, It's a FQDN – eg `docs.mycompany.com`